### PR TITLE
Add set

### PR DESCRIPTION
**Pull request by abadabada**
Adding set to avoid repetition of words.
Adding all words from the wordsList[] to wordsListSet<>
Adding random 25 words from wordsListSet<>to wordsToDisplay[]

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (latedude2) [inline .vscode/launch.json:1]: This whole file should be added to gitignore as it is API specific

File: src/server/WordGenerator.java [hunk 2]
```diff
@@ -3,6 +3,10 @@
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.io.IOException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
 
 public class WordGenerator{
```
Comment (latedude2) [inline src/server/WordGenerator.java]: I think this class should be static as it has no constructor and is never initialised.

File: src/server/WordGenerator.java [hunk 3]
```diff
@@ -21,8 +25,15 @@ public void readFile(){
...
-            for(int i = 0; i < 25; i++){
-                wordsToDisplay[i] = wordList[getRandom(968)];
+            Set<String> wordListSet = new HashSet<>();
+            Collections.addAll(wordListSet, wordList);
+            int size = wordListSet.size();
+            int item = new Random().nextInt(size);
+            
+            for(int j = 0; j < 25; j++)
+            {
+                    wordsToDisplay[j] = wordListSet.toArray(new String[wordListSet.size()])[item];
```
Comment (latedude2) [inline src/server/WordGenerator.java]: Fix tabbing on this line pls. Also, you can replace wordListSet.size() with size here.

**Review by latedude2 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (abadabada) [reply .vscode/launch.json:1]: it is in gitignore already idk why it is doing this.

File: src/server/WordGenerator.java (hunk 2 above)
Comment (abadabada) [reply src/server/WordGenerator.java]: Making a class classic means it should be another class, does that mean I have to move it to main and creat a static one there?

**Review by latedude2 — APPROVED** [review]

**Pull request merged**